ministryofjustice/correspondence_tool_staff
# CT-3017 changed the conditions of showing general close report based on permission

## Description

This PR  is to "hide" the general-closed-cases-report for offender-sar user as Offender-SAR has own version of closed-cases-report which is under 'Offender-SAR' type.  The method I used is to hide them by checking the user's permission.   I also changed some flags for 'R007' in DB in order to achieve it

## Self-review checklist


### Screenshots

Before 
<img width="626" alt="Screenshot 2020-09-09 at 10 11 02" src="https://user-images.githubusercontent.com/26111671/92579914-b0dbfa00-f285-11ea-83a2-2501f9ca0c6b.png">

after
<img width="565" alt="Screenshot 2020-09-09 at 10 14 24" src="https://user-images.githubusercontent.com/26111671/92579943-bafdf880-f285-11ea-8469-a1958d6832a7.png">

for FOI / SAR type, nothing change 
<img width="677" alt="Screenshot 2020-09-09 at 10 11 22" src="https://user-images.githubusercontent.com/26111671/92579976-c51ff700-f285-11ea-935b-00bcc2581798.png">

 
### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3017
### Deployment

Please run data migrate command 
rails data:migrate
### Manual testing instructions

1. login as branston user
2. click "Reports" tab
3. click "Create report"
4, you should only see 'Offender SAR'
5. sign out
6. login as london user
7. repeat 2, 3 step
8. you should see 'FOI', 'SAR' and 'General closed cases report'
